client/package.json

```diff
@@ -10,14 +10,17 @@
         "@types/blessed": "^0.1.17",
         "babel-jest": "^27.0.6",
         "blessed": "^0.1.81",
+        "dotenv": "^10.0.0",
         "eslint": "^7.29.0",
         "isomorphic-fetch": "^3.0.0",
         "nodemon": "^2.0.9",
         "ts-node": "^10.0.0",
         "typescript": "^4.3.5"
     },
     "scripts": {
-        "start": "ts-node src/index.ts",
+        "start": "NODE_ENV=local ts-node src/index.ts",
```

Comment:
`local` is not a valid value for `NODE_ENV`. Use `development`. Why do you need the `local` for?